packages/cli/src/commands/server/middleware/logToConsoleMiddleware.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+import chalk from 'chalk';
+
+let cache = [];
+let timer;
+
+const log = ({level, data, id}) => {
```

Comment:
You should probably use our custom logger.

Reply:
What's the custom logger and what does it do? This basically just forwards `console` calls from the device onto the terminal without any special magic except Jest-like highlights.

Reply:
We have those Jest-like highlight already implemented. https://github.com/react-native-community/cli/blob/master/packages/tools/src/logger.ts

Reply:
I believe it would be this: https://github.com/react-native-community/cli/blob/master/packages/tools/src/logger.ts

Reply:
I don't really think this adds any value here. The current logger is highlighting information issued by the CLI, and here we are highlighting information from an RN app. I think it makes sense for those to look differently because one is from the tool and the other from the user. A different style will make people pay attention differently.

Reply:
Agree with @cpojer on that. We're gonna overhaul the UI Metro gives us some time later, but not today.